Sources/HTTP/PoCSocket/PoCSocket.swift

```diff
@@ -200,14 +200,14 @@ internal class PoCSocket {
         #if os(Linux)
             var addr = sockaddr_in(
                 sin_family: sa_family_t(AF_INET),
-                sin_port: UInt16(port),
+                sin_port: htons(UInt16(port)),
```

Comment:
I think you can use `UInt16(bigEndian:)` (and below).

Reply:
@weissi is this the same thing as you did in https://github.com/apple/swift-corelibs-foundation/pull/1185 ?

Reply:
The use of `htons()` on Linux for host to network byte order conversion is pretty standard - and probably makes it clearer what the code is doing here. The non-Linux case is more opaque.

Reply:
@ianpartridge do you have any major concerns merging as-is? We can always revisit `htons` vs. alternatives later.

Reply:
No major concerns, I was suggesting a possible improvement which we can always make in future.

Reply:
@ianpartridge / @seabaylea `htons()` is the right thing to use here. The only `UInt16(bigEndian:)` is the wrong way around (usually called `ntohs()`). The only reason I used it and not `ntohs()` is that `ntohs` is a C macro that's not imported into Swift.

Reply:
you could use `UInt16(port).bigEndian` but `htons()` is to be preferred as it IMHO carries more semantic meaning.